src/k8s-extension/azext_k8s_extension/tests/latest/data/cert_and_key_encoded.txt

```diff
@@ -0,0 +1,2 @@
+dGVzdGNlcnQ=
```

Comment:
Can you please put these files in extension-specific directories such as `src/k8s-extension/azext_k8s_extension/tests/latest/partner_extensions/azure_ml/data/...`

Reply:
tbh, I would actually reverse the order and do data/azure_ml, otherwise it might be confusing for the future, since azureml related test are under src/k8s-extension/azext_k8s_extension/tests/latest/partner_extensions/public/ Hence I think it makes sense to introduce a data director that is going to have extension specific sub directories. Thoughts?